cleanupJavadoc.sh

```diff
@@ -1,2 +0,0 @@
-find release/doc -name "*.html" -exec sed -i "s/.*Generated by javadoc.*//" {} \;
```

Comment:
Actually I'm wondering if it is a good idea to keep the generated javadoc in the GIT repo. I like the idea of checking out and having the doc ready on your hard drive, but on the other hand it is generated stuff, so it has little value.